# HG changeset patch
# User sbr <sebastien@brissaud.name>
# Date 1375042888 -7200
#      Sun Jul 28 22:21:28 2013 +0200
# Node ID 8127c5e6272cfa417d7a63e66b6f5bad12223e89
# Parent  5bd2929788796a4f9874dcb74eb65d56200fd5dd
-Cleanup: Use an empty DropdownListItem as seperator instead of an empty string

diff --git a/src/lang/english.txt b/src/lang/english.txt
--- a/src/lang/english.txt
+++ b/src/lang/english.txt
@@ -457,7 +457,6 @@ STR_NEWS_MENU_MESSAGE_HISTORY_MENU      
 
 ############ range for about menu starts
 STR_ABOUT_MENU_LAND_BLOCK_INFO                                  :Land area information
-STR_ABOUT_MENU_SEPARATOR                                        :
 STR_ABOUT_MENU_TOGGLE_CONSOLE                                   :Toggle console
 STR_ABOUT_MENU_AI_DEBUG                                         :AI/Game script debug
 STR_ABOUT_MENU_SCREENSHOT                                       :Screenshot
diff --git a/src/toolbar_gui.cpp b/src/toolbar_gui.cpp
--- a/src/toolbar_gui.cpp
+++ b/src/toolbar_gui.cpp
@@ -1037,7 +1037,7 @@ static CallBackFunction ToolbarHelpClick
 {
 	DropDownList *list = new DropDownList();
 	list->push_back(new DropDownListStringItem(STR_ABOUT_MENU_LAND_BLOCK_INFO,        HME_LAND_BLOCK_INFO, false));
-	list->push_back(new DropDownListStringItem(STR_ABOUT_MENU_SEPARATOR,              -1, false));
+	list->push_back(new DropDownListItem(-1, false)); // separator
 	list->push_back(new DropDownListStringItem(STR_ABOUT_MENU_TOGGLE_CONSOLE,         HME_TOGGLE_CONSOLE, false));
 	list->push_back(new DropDownListStringItem(STR_ABOUT_MENU_AI_DEBUG,               HME_AI_DEBUG, false));
 	list->push_back(new DropDownListStringItem(STR_ABOUT_MENU_SCREENSHOT,             HME_SCREENSHOT, false));
